RELEASE-NOTES.md

```diff
@@ -1,5 +1,39 @@
 # Prefect Release Notes
 
+## Release 2.14.2
+
+## Ability to pass **kwargs to `on_completion` and `on_failure` hooks
```

Comment:
you should now be able to pass partials as any hook for tasks and flows, unless there's some reason to avoid mentioning `on_cancellation` here
```suggestion
## Ability to pass **kwargs to state change hooks
```